.chloggen/cloudwatch-metric-benchmark.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awscloudwatchmetricstreamsencodingextension
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add benchmark test to JSON unmarshaler.
```

Comment:
I don't think a changelog is needed since it has no incidence on users or the api.

Reply:
Removed